test/gtest/ucs/test_bitops.cc

```diff
@@ -90,112 +90,52 @@ UCS_TEST_F(test_bitops, ptr_ctz) {
     ASSERT_EQ(88, ucs_count_ptr_trailing_zero_bits(buffer, 160));
 }
 
+void check_bitwise_equality(const uint8_t *buffer1,
```

Comment:
can it be static function inside test_bitops?

Reply:
yes, done in "PR fixes 3.0", the reason for the stack size here is `ucs_bitwise_is_equal()` function, which is always inlined